.travis_do.sh

```diff
@@ -102,6 +102,44 @@ EOF
 	exit $RET
 }
 
+test_commits() {
+	RET=0
+	for commit in $(git rev-list $TRAVIS_COMMIT_RANGE); do
+		echo_blue "=== Checking commit '$commit'"
+		if git show --format='%P' -s $commit | grep -qF ' '; then
+			echo_red "Pull request should not include merge commits"
+			RET=1
+		fi
+
+		author=$(git show -s --format=%aN $commit)
+		if echo $author | grep -q '\S\+\s\+\S\+'; then
+			echo_green "Author name ($author) seems ok"
+		else
+			echo_red "Author name ($author) need to be your real name 'firstname lastname'"
+			RET=1
+		fi
+
+		subject=$(git show -s --format=%s $commit)
+		if echo $subject | grep -q '^[0-9A-Za-z,]\+: '; then
+			echo_green "Commit subject line seems ok ($subject)"
+		else
+			echo_red "Commit subject line MUST start with '<package name>: ' ($subject)"
+			RET=1
+		fi
+
+		body=$(git show -s --format=%s $commit)
```

Comment:
in theory yes, but I just saw a typo (```%s``` instead of ```%b```) so some more testing required